views/overlays/finalsBracket.hbs

```diff
@@ -129,14 +138,26 @@
         div.innerHTML = '';
       });
 
-      top8.matches.forEach(function(match) {
-        var round = match.round < 0 ? match.round + top8round + losersOffset : match.round - top8round;
+      matches.forEach(function(match) {
+        var round = match.round < 0 ? match.round - minRound - 4 : match.round - maxRound + 3;
```

Comment:
Could you possibly add a comment around this logic? I haven't looked much into bracket logic so this formula isn't exactly intuitive.